### ISSUE TITLE
  Errors in one's own model

### ISSUE THREAD

**Issue by Anoue** [wang-xinyu/tensorrtx#1398]
## Env

- GPU, e.g. V100
- OS, e.g. Ubuntu22.04
- Cuda version 11.3
- TensorRT version 8.2.1

## Your problem

Why was the engine file successfully generated, but there were no results during detection?

**Comment by wang-xinyu** [comment]
Which model? Did you try the official model?

**Comment by Anoue** [comment]
> Which model? Did you try the official model?
I trained my own model and used YOLOv5s as the benchmark network.

Using official weights, no problem

And then I used softnms for nms, which is not compatible.

I think other network structures have been adapted, and there were no errors reported during the process of generating the engine file.

So I don't know if there is a problem with softnms. Have you adapted to softnms?

**Comment by Anoue** [comment]
> Which model? Did you try the official model?

I trained my own model and used YOLOv5s as the benchmark network.

Using official weights, no problem

And then I used softnms for nms, which is not compatible.

I think other network structures have been adapted, and there were no errors reported during the process of generating the engine file.

So I don't know if there is a problem with softnms. Have you adapted to softnms?
